testing/unittests/unittests.cpp

```diff
@@ -124,7 +125,7 @@ int main(int argc, char* argv[])
     bool includeAll = false;
     bool verbose = false;
     bool list = false;
-    bool useDefaultLocations = true;
+    bool useDefaultLocations = false;
```

Comment:
@afishbeck was this change intentional?

Reply:
@ghalliday oops that was to simplify my testing.  I'll remove.